### DISCUSSION TITLE
problems with changing speed for sparse infill

### DISCUSSION THREAD

**Discussion by JorianRennen** [OrcaSlicer/OrcaSlicer#2944] [Q&A]
Hi there, 

I'm new to orcaslicer and I have this issue regarding the speed of the sparse infill.
When I try to change the printing speed of my model, i noticed that my infill speed is the same as the outer and innerwall, while my settings say otherwise. Anyone an idea why this is? Am I missing a setting? 

![image](https://github.com/SoftFever/OrcaSlicer/assets/80458086/800963ce-881e-4822-a8cb-5b636cbae118)
![image](https://github.com/SoftFever/OrcaSlicer/assets/80458086/a5dcef40-1bdf-41d1-8dc2-97927d11f379)
As you can see, both inner/outerwall and infill speed are about 70 mm/s

Here are the settings:
![image](https://github.com/SoftFever/OrcaSlicer/assets/80458086/19e13329-a460-40e3-aac6-6f62d24226f3)
And the settings of my printer:
![image](https://github.com/SoftFever/OrcaSlicer/assets/80458086/9fed8e7c-2da6-443a-97bf-929778ac87e5)

I feel like I'm missing something, but I really don't know what...

Thanks in advance!

**Comment by ffgtthr** [comment]
Check minimum layer times in filament settings>cooling.
![image](https://github.com/SoftFever/OrcaSlicer/assets/117250863/ecba9f3e-14e3-4f6a-bb79-c7ab0f84a14c)
Also check max volumetric speed in filament settings>filament.
![image](https://github.com/SoftFever/OrcaSlicer/assets/117250863/be2cee48-5e4a-41be-943b-68f73004045c)

**Reply by mikeq3** [reply]
You have to change your setting in the filament for max layer time that is next to "Max fan speed threshold". If you lower the value for Layer time it will change things. Read the description for that box.

**Comment by vgdh** [comment]
maybe you hit max volumetric flow?
